src/Transfer.php

```diff
@@ -3990,8 +3990,8 @@ class='btn btn-primary'></div>";
         echo "</td></tr>";
 
         echo "<tr class='tab_bg_1'>";
-        echo "<td>" . __('Disable locked field during transfer') . "</td><td>";
-        Dropdown::showYesNo('locked_field', $this->fields['locked_field']);
+        echo "<td>" . __('Lock fields updated during transfer') . "</td><td>";
+        Dropdown::showYesNo('lock_updated_fields', $this->fields['lock_updated_fields']);
```

Comment:
Unless I've missed something, this is no longer the same name as the field in the database

Reply:
should be fixed